sh.shuriken.Eeman.json

```
@@ -0,0 +1,100 @@
+{
+    "id" : "sh.shuriken.Eeman",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "eeman",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=session-bus"
```

```suggestion
        "--socket=wayland"
```

Shouldn't be needed anymore